### PR TITLE
Installation script update

### PR DESCRIPTION
**Pull request by ghost**
Installation script update

Addresses: https://github.com/dotnet/sdk/issues/3827

### CODE REVIEW

File: scripts/obtain/dotnet-install.sh [hunk 1]
```diff
@@ -255,7 +255,7 @@ check_pre_reqs() {
         local librarypath=${LD_LIBRARY_PATH:-}
         LDCONFIG_COMMAND="$LDCONFIG_COMMAND -NXv ${librarypath//:/ }"
 
-        [ -z "$($LDCONFIG_COMMAND 2>/dev/null | grep libunwind)" ] && say_warning "Unable to locate libunwind. Probable prerequisite missing; install libunwind."
+        [ -z "$($LDCONFIG_COMMAND 2>/dev/null | grep zlib)" ] && say_warning "Unable to locate zlib. Probable prerequisite missing; please install zlib."
```
Comment (livarcocc) [inline scripts/obtain/dotnet-install.sh]: were we missing zlib before? the warning that is.

Comment (johnbeisner) [reply scripts/obtain/dotnet-install.sh]: It appears that we were missing 'zlib' before...
See: https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=netcore30#supported-linux-versions

**Review by livarcocc — APPROVED** [review]
I marked this approved. But shouldn't this go to dotnet/toolset?

dotnet/cli @ master is being deprecated and the dotnet/cli and dotnet/toolset (in progress) repos are going to be merged into dotnet/sdk.

**Comment by johnbeisner** [comment]
@livarcocc 
You are correct, this should go into toolset. However, there are 2 active PRs against CLI: https://github.com/dotnet/cli/pull/12880 and https://github.com/dotnet/cli/pull/12642 that I am driving to completion - we will finish up in CLI then I will merge all changes once into toolset.

**Comment by nguerrera** [comment]
Please make sure to preserve attribution when porting to toolset

**Pull request merged**